src/utils/navigation.ts

```diff
@@ -77,7 +77,7 @@ export const NAVIGATION: Navigation = {
   [config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL]: {
     currentPage: config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
     previousPage: (appData: ApplicationData, req: Request) => getUpdateOrRemoveBackLink(req, config.OVERSEAS_ENTITY_QUERY_URL),
-    nextPage: [config.UPDATE_FILING_DATE_PAGE]
+    nextPage: [config.UPDATE_FILING_DATE_PAGE, config.PRESENTER_URL]
```

Comment:
I wouldn't change this. I don't think `nextPage` is currently used anyway, so doesn't achieve anything, but if it did, the new value wouldn't be right for the update journey (who's next page is still the update filing date page).

Reply:
Removed change from PR.